pylds/discrete_maps.py

```diff
@@ -0,0 +1,130 @@
+
+import numpy as np
+
+def StandardMap(u_initial, PARAMETERS=[0.3]):
+    """
+    Returns 1D Hamilton-Centre vector field at time t, for an array of points in phase space.
+    Number of model parameters: 1 . PARAMETERS = [omega]
+    Functional form: v = (omega*y, - omega*x), with u = (x, y)
+    
+    Parameters
+    ----------
+    u : array_like, shape(n,)
+        points in phase space to determine vector field at time t
+    
+    PARAMETERS : list of floats
+        vector field parameters
+    
+    Returns
+    -------
+    v : array_like, shape(n,)
+        vector field corresponding to points u, in phase space at time t
+    """
+    x_initial, y_initial = u_initial.T
+    # Hamiltonian Model Parameter
+    K, = PARAMETERS
+    
+    # Map components 
+    x_next = x_initial + y_initial - (K/(2*np.pi))*np.sin(2*np.pi*x_initial)
+    y_next = y_initial - (K/(2*np.pi))*np.sin(2*np.pi*x_initial)
+    
+    # Map next iteration
+    u_next = np.array([ x_next, y_next]).T
```

Comment:
`np.column_stack`

Reply:
What's up with this? This is the thrid of my comments you marked as resolved without addressing it.

Reply:
Same as above.

Reply:
Line 32 is unchanged online.